Think of texts as arks.
Each text invariably contains a genetic code if you consider the letters T, G, A and C as representative initials of Thymine, Guanine, Adenine, and Cytosine.
This short program extracts each of those letters from a properly prepared text. (You'll probably want to capitalize and remove all punctuation and spaces to make it less prone to throwing errors.)
Once you've a viable genetic sample sequence (likely several thousand characters long), you can enter it into a genetic database to determine which real organisms share genetic materical with that text.
For example, Charles Darwin's _On the Origin of Species_ shares ranges of DNA with the _Epinotia Bilunana_ moth, specifically chromosomes 17 and 10.
Three publicly accessible databases store large amounts of nucleotide and protein sequence data: GenBank (BLAST alignment tool) at the National Center for Biotechnology Information (NCBI) of the National Institutes of Health (NIH), the DNA Database of Japan (DDBJ), and the European Bioinformatics Institute (EBI)'s EMBL database tool.
You may also generate the corresponding helix or the RNA sequences corresponding to the genetic sample generated. With RNA, each T (Thymine) nucleotide would be replaced by a U (Uracil).
The databases also allow for protein sequence searches. Grouping an RNA sequence three-by-three produces codons which correspond to proteins. Charts are available for reference on Google.
Free full-length texts are also available on Project Gutenberg website.
Godspeed, ladies, gentlemen and everyone in between.

P.S. You may have different results depending on which browser you use (e.g. Google Chrome vs. Tor Browser).
